open_fdd/desktop_bridge/server.py

```
@@ -173,58 +343,117 @@ def _disk_info() -> dict[str, int | float | str]:
+        "onboard": _driver_health_entry(),
+    }
+
+    async def _ttl_sync_loop(app_ref: FastAPI) -> None:
         while True:
             try:
                 path = services.ttl.sync()
-                app.state.last_ttl_sync_iso = str(path)
-                app.state.ttl_sync_error = ""
+                app_ref.state.last_ttl_sync_iso = str(path)
+                app_ref.state.ttl_sync_error = ""
             except Exception as exc:  # pragma: no cover - defensive runtime loop
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)